src/textual/app.py

```diff
@@ -1290,23 +1322,64 @@ def _on_idle(self) -> None:
             self._require_stylesheet_update.clear()
             self.stylesheet.update_nodes(nodes, animate=True)
 
-    def _register_child(self, parent: DOMNode, child: Widget) -> bool:
+    def _register_child(
+        self, parent: DOMNode, child: Widget, before: int | None, after: int | None
+    ) -> bool:
```

Comment:
Docstring please (still like to document 'private' methods. I'm aware that the original version didn't have a docstring...

Reply:
Added. Also drop the return value as it is never used by the single instance of a call.